src/main/java/teammates/storage/entity/FeedbackQuestion.java

```diff
@@ -44,9 +45,15 @@
     // TODO: Do we need this field since creator of FS = creator of qn? (can be removed -damith)
```

Comment:
Another ancient, unaddressed comment?

Reply:
Tracked in the project https://github.com/TEAMMATES/teammates/projects/5#card-13088552